src/backend/rasa/data/nlu.yml

```diff
@@ -32,7 +32,7 @@ nlu:
       - goodevening
       - good
   
-  - intent: introduce_myself
+  - intent: utter_introduction
```

Comment:
I think `utter` is only supposed to go before bot responses if I'm not mistaken. But probably wouldn't cause any issues so probably don't need to change